pfs_middleware/pfs_middleware/middleware.py

```diff
@@ -1829,6 +1932,7 @@ def head_object(self, ctx):
             rpc.parse_head_response(head_response)
 
         headers = swob.HeaderKeyDict(deserialize_metadata(raw_md))
+        unmung_etags(headers, num_writes)
```

Comment:
This works equally-well with both `dict`s and `HeaderKeyDict`s, right?

Reply:
i have no idea!  i will need to investigate ...